src/Faker/Provider/sv_SE/Address.php

```diff
@@ -7,15 +7,15 @@ class Address extends \Faker\Provider\Address
     protected static $buildingNumber = array('%###', '%##', '%#', '%#?', '%', '%?');
 
     protected static $streetPrefix = array(
-        'Stor', 'Små', 'Lill', 'Sjö', 'Kungs', 'Drottning', 'Hamn', 'Brunns', 'Linné', 'Vasa', 'Ring', 'Freds'
+        'Björk','Järnvägs','Ring','Skol','Skogs','Ny','Gran','Idrotts','Stor','Kyrk','Industri','Park','Strand','Skol','Trädgårds','Industri','Ängs','Kyrko','Park','Villa','Ek','Kvarn','Stations','Back','Furu','Gen','Fabriks','Åker','Bäck','Stor','Små','Lill','Sjö','Kungs','Drottning','Hamn','Brunns','Linné','Vasa','Ring','Freds'
```

Comment:
where did you take that data from? Why is it more relevant than the previous one? Also, please sort terms alphabetically.